src/main/java/io/jenkins/plugins/synopsys/security/scan/extension/freestyle/SecurityScanFreestyle.java

```diff
@@ -432,6 +467,21 @@ public void setBlackduck_reports_sarif_severities(String blackduck_reports_sarif
         this.blackduck_reports_sarif_severities = Util.fixEmptyAndTrim(blackduck_reports_sarif_severities);
     }
 
+    @DataBoundSetter
+    public void setBlackduck_search_depth(Integer blackduck_search_depth) {
+        this.blackduck_search_depth = blackduck_search_depth;
```

Comment:
We should use `Util.fixEmptyAndTrim()` method in case of String types. Look into other new setters methods too. I see that we are trimming off before putting these values into the parameters map (in the ScanParametersMap class). So we should trim it here.

Reply:
This is an **integer** field

Reply:
Then we do not need these for Integers. But check for String